runtime/src/main/java/com/redhat/ipaas/rest/BaseHandler.java

```diff
@@ -0,0 +1,15 @@
+package com.redhat.ipaas.rest;
+
+import javax.inject.Inject;
+
+public class BaseHandler implements WithDataManager {
+
+    @Inject
+    private DataManager dataMgr;
+
+    @Override
+    public DataManager dataManager() {
```

Comment:
fwiw for the model objects the ReflectiveSorter relies on having get-methods (but could be adapted if one feels so)